src/cider/nrepl/middleware/complete.clj

```diff
@@ -5,16 +5,22 @@
    [cljs-tooling.complete :as cljs-complete]
    [compliment.core :as jvm-complete]
    [compliment.utils :as jvm-complete-utils]
-   [orchard.misc :as u]))
+   [orchard.misc :as u]
+   [suitable.complete-for-nrepl :as suitable]))
+
+(defn- cljs-complete
+  [msg cljs-env ns prefix extra-metadata]
+  (concat (cljs-complete/completions cljs-env prefix {:context-ns ns
+                                                      :extra-metadata extra-metadata})
+          (suitable/complete-for-nrepl msg)))
```

Comment:
Why does `suitable` need the entire `msg`? I'm asking mostly because it seems to me that all the other params should be enough to get candidates.

Reply:
It needs access to the cljs options + repl env + compiler env. We can pull those out and just pass them along but it seemed simpler to hand in the msg.

Reply:
Got it. As this is something minor I guess there's no need to fret over it too much.